src/templates/layouts/healthCareLocalFacility/Phone.tsx

```diff
@@ -0,0 +1,154 @@
+import { ParagraphPhoneNumber } from '@/types/drupal/paragraph'
+
+export interface SeparatedPhoneNumber {
+  phoneNumber: string
+  extension: string
+  processed: boolean
+}
+
+export const separatePhoneNumberExtension = (
+  phoneNumber: string
+): SeparatedPhoneNumber | null => {
+  if (!phoneNumber) {
+    return null
+  }
+
+  const phoneRegex =
+    /\(?(\d{3})\)?[- ]*(\d{3})[- ]*(\d{4}),?(?: ?x\.? ?(\d*)| ?ext\.? ?(\d*))?(?!([^<]*>)|(((?!<v?a).)*<\/v?a.*>))/i
+
+  const match = phoneRegex.exec(phoneNumber)
+
+  if (!match || !match[1] || !match[2] || !match[3]) {
+    // Short number, invalid format, or not a full US number
+    return {
+      phoneNumber,
+      extension: '',
+      processed: false,
+    }
+  }
+
+  const phone = `${match[1]}-${match[2]}-${match[3]}`
+  const extension = match[4] || match[5] || ''
+
+  return {
+    phoneNumber: phone,
+    extension,
+    processed: true,
+  }
+}
+
+export const processPhoneToVaTelephoneOrFallback = (
+  phoneNumber: string = ''
+) => {
+  const internationalPattern = /\(?(\+1)\)?[- ]?/gi
+
+  if (!phoneNumber) {
+    return null
+  }
+
+  const separated = separatePhoneNumberExtension(phoneNumber)
+
+  if (separated.processed) {
+    return (
+      <va-telephone
+        contact={separated.phoneNumber}
+        extension={separated.extension}
+        international={internationalPattern.test(phoneNumber)}
+      />
+    )
+  }
+
+  return <a href={`tel:+1${phoneNumber}`}>{phoneNumber}</a>
+}
+
+export const PhoneNumberFromField = ({
+  phoneNumber,
+  phoneExtension,
+  phoneNumberType,
+  phoneLabel,
+}: {
+  phoneNumber: string
+  phoneExtension?: string
+  phoneNumberType?: string
+  phoneLabel?: string
+}) => {
+  if (!phoneNumber) return null
+
+  const isSms = phoneNumberType === 'sms'
+  const isTty = phoneNumberType === 'tty'
+  const isFax = phoneNumberType === 'fax'
+
+  let partialPhoneNumber = phoneNumber
+  let partialPhoneExtension = phoneExtension
+
+  if (!phoneExtension) {
+    const separated = separatePhoneNumberExtension(phoneNumber)
+    partialPhoneNumber = separated.phoneNumber
+    partialPhoneExtension = separated.extension
+  }
+
+  return (
+    <p className="vads-u-margin-bottom--0 vads-u-margin-top--0">
+      <strong>{phoneLabel || 'Mental health care'}: </strong>
```

Comment:
Is it always Mental health care if no label?

Reply:
[This is what I was looking at.](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/layouts/health_care_local_facility.drupal.liquid#L121-L130)